test/multibuild_test.js

```diff
@@ -2072,4 +2072,49 @@ describe("multi build", function(){
 				});
 			});
 	});
+
+	it("does not swallow clean-css errors", function(done) {
+		var base = path.join(__dirname, "broken_css_build");
+
+		asap(rmdir)(path.join(base, "dist"))
+			.then(function() {
+				return multiBuild({
+					main: "main",
+					config: path.join(base, "stealconfig.js")
+				}, {
+					quiet: true,
+					cleanCSSOptions: {
+						inline: ["all"] // set inline @import to trigger build error
+					}
+				});
+			})
+			.then(function() {
+				assert.ok(false, "build promise should not resolve");
+				done();
+			})
+			.catch(function(err) {
+				assert.ok(!!err, "build promise should be rejected");
+				done();
+			});
+	});
+
+	it("css imports are not inlined by default", function() {
+		var base = path.join(__dirname, "css_imports");
+
+		return asap(rmdir)(path.join(base, "dist"))
+			.then(function() {
+				return multiBuild({
+					main: "main",
+					config: path.join(base, "stealconfig.js")
+				}, {
+					quiet: true
+				});
+			})
+			.then(function() {
+				var source = fs.readFileSync(path.join(base, "dist", "bundles",
```

Comment:
don't use `@import` then, it will cost you a request in production, load it through steal.... If you think this is a problem please file and issue and we'll discuss it there.